src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
+  let totalAmount = 0;
+  let discount;
+  const rentAmount = 40;
+
   // write code here
+  if (days < 3) {
+    discount = 0;
+  } else if (days < 7) {
+    discount = 20;
+  } else {
+    discount = 50;
+  }
```

Comment:
```suggestion
  if (days < 3) {
    return days * rentAmount
  }
  
  if (days < 7) {
    return (days * rentAmount) - smallDiscount
  }
  
  return (days * rentAmount) - bigDiscount
```
 Упрости эту конструкцию, вынеси 20 и 50 в переменные

Reply:
Спасибо, переделал. Использование if без else if сугубо для упрощения? вообще используют if/else if на практике?

Reply:
используют, но реже. В большинстве случаев хватает просто нескольких if